### PR TITLE
Count games even before it's our move.

### PR DESCRIPTION
**Pull request by Dorus**

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -505,10 +500,10 @@ class Connection {
...
     }
-    processMove(gamedata) {
-        const game = this.connectToGame(gamedata.id)
-        game.makeMove(gamedata.move_number);
-    }
+    // processMove(gamedata) {
+    //     const game = this.connectToGame(gamedata.id)
+    //     game.makeMove(gamedata.move_number);
+    // }
     processStoneRemoval(gamedata) {
```
Comment (wonderingabout) [inline connection.js:507]: unless i'm mistaken you can also delete or comment out processStoneRemoval

Comment (wonderingabout) [reply connection.js:507]: on_friendRequest also seems unused:
https://github.com/Dorus/gtp2ogs/blob/CountActiveGames/connection.js#L260-L266

Comment (Dorus) [reply connection.js:507]: Indeed, so many unused functions here. Same with game started method. And delete too if i add it to the ignore list on top.

I think i'm going to make another pr for delete notifications and add this there.

Comment (wonderingabout) [reply connection.js:507]: yes, making a separate pr would be better, here we're just fixing the connectedgames problem

Comment (wonderingabout) [reply connection.js:507]: you may want to put back these lines as they were, as i made a specific pr #266

**Comment by roy7** [comment]
@Dorus This good to merge?

**Pull request merged**